java-backend/nativeServer/src/main/java/com/github/cambridgeAlphaTeam/Server.java

```diff
@@ -34,8 +41,56 @@
 
     private static OscSender sender;
     private static final Logger logger = LoggerFactory.getLogger(Server.class);
+    /* Command line options parser */
+    private static final Options options = new Options();
+    static {
+        /* addOption(String short option,
+         *           String long option,
+         *           boolean has arguments,
+         *           String description)
+         *
+         * where a short option is like "-h" and a long option is like
+         * "--help" (which takes no arguments). The description is for
+         * generating the help message.
+         */
+        options.addOption ("h", "help", false, "Print this help message");
+        options.addOption ("u", "usage", false, "Print the list of options");
+        options.addOption ("c", "cubelets-program", true, "A single string of the cubelet program. " +
+                "If you want to pass arguments to it, make a batch/shell file and put the arguments to that file, " +
```

Comment:
How do you delimit those, i.e. how would the parser know when to stop parsing it as cubelet options.